Template.php

```diff
@@ -654,6 +654,7 @@ protected function get_identifiers_from_url($url_sent = NULL) {
         $ch = curl_init($encoded_url);
         curl_setopt($ch, CURLOPT_HEADER, 1);
         curl_setopt($ch, CURLOPT_NOBODY, 1);
+        curl_setopt($ch, CURLOPT_VERBOSE, 0); // stop curl from printing non-errors to stdout
```

Comment:
Ought we to (or can we) set this option globally rather than repeatedly?

Reply:
I do not know.  It is only actually needed here, since this is the only place in the code that we have curl return TRUE/FALSE instead of an object.